### PR TITLE
pull_request_template.md: fix link to CONTRIBUTING

### PR DESCRIPTION
**Pull request by mipmip**
Before submitting a pull request, please read [CONTRIBUTING](../CONTRIBUTING.md); pull requests that do not meet the criteria described there will not be merged. Note that this repository's CONTRIBUTING contains information specific to this repository, and is not the same as CONTRIBUTING for the void-packages repository.

We prioritise pull requests involving information specific to Void over those involving information applicable to Linux in general.

### CODE REVIEW

**Comment by ericonr** [comment]
I think it should be `https://github.com/void-linux/void-docs/blob/master/CONTRIBUTING.md`. The new link still can't be clicked.

**Comment by mipmip** [comment]
Absolute link is most safe. I changed it.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-Before submitting a pull request, please read [CONTRIBUTING](./CONTRIBUTING.md); pull requests that do not meet the criteria described there will not be merged. Note that this repository's CONTRIBUTING contains information specific to this repository, and is not the same as CONTRIBUTING for the void-packages repository.
+Before submitting a pull request, please read [CONTRIBUTING](../CONTRIBUTING.md); pull requests that do not meet the criteria described there will not be merged. Note that this repository's CONTRIBUTING contains information specific to this repository, and is not the same as CONTRIBUTING for the void-packages repository.
```
Comment (ericonr) [inline .github/pull_request_template.md:1]: Are you sure you pushed the correct commit?

Comment (mipmip) [reply .github/pull_request_template.md]: I did it wrong.

**Comment by ericonr** [comment]
Thanks!

**Pull request merged**